2022-06-22 23:14:55,213 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:46]
2022-06-22 23:15:20,613 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:46]
2022-06-22 23:20:13,264 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:46]
2022-06-22 23:22:37,351 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:46]
2022-06-22 23:24:45,991 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:46]
2022-06-22 23:26:00,723 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:46]
2022-06-22 23:26:59,988 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:46]
2022-06-22 23:27:48,087 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:46]
2022-06-22 23:27:49,977 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:46]
2022-06-22 23:34:15,474 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:46]
2022-06-22 23:35:03,315 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:46]
2022-06-22 23:35:47,070 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:46]
2022-06-22 23:40:13,214 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:46]
2022-06-22 23:47:32,987 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:47]
2022-06-23 00:04:53,593 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:47]
2022-06-23 00:11:34,766 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:47]
2022-06-23 00:12:56,204 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:47]
2022-06-23 00:35:09,143 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:47]
2022-06-23 00:35:11,080 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:47]
2022-06-23 00:39:07,366 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:47]
2022-06-23 00:41:42,664 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:47]
2022-06-23 00:42:10,916 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:47]
2022-06-23 00:42:26,094 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:47]
2022-06-23 00:52:23,261 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\errors\handlers.py:47]
2022-06-23 17:34:16,033 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:70]
2022-06-23 17:34:17,938 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:70]
2022-06-23 17:36:06,918 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:70]
2022-06-23 17:37:42,319 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:70]
2022-06-23 17:43:59,724 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:71]
2022-06-23 17:44:25,828 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:71]
2022-06-23 17:44:27,954 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:71]
2022-06-23 21:50:53,642 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 21:50:55,579 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 22:03:43,944 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 22:08:44,172 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 22:18:51,310 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 22:18:54,270 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 22:28:01,627 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 22:28:03,657 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 22:33:39,125 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 22:34:04,416 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 22:35:29,112 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 22:45:37,921 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 22:47:14,787 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 22:49:07,045 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 22:57:36,685 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 23:02:28,975 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 23:02:32,302 ERROR: Exception on /user/user/VadimM [GET] [in C:\Users\zayaz\PycharmProjects\Customer-service\venv\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "C:\Users\zayaz\PycharmProjects\Customer-service\venv\lib\site-packages\flask\app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\zayaz\PycharmProjects\Customer-service\venv\lib\site-packages\flask\app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\zayaz\PycharmProjects\Customer-service\venv\lib\site-packages\flask\app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\zayaz\PycharmProjects\Customer-service\venv\lib\site-packages\flask\app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\zayaz\PycharmProjects\Customer-service\venv\lib\site-packages\flask_login\utils.py", line 303, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
  File "C:\Users\zayaz\PycharmProjects\Customer-service\app\user\routes.py", line 35, in user
    return render_template('user.html', user=user, title=username, form=form, profile_form=profile_form,
  File "C:\Users\zayaz\PycharmProjects\Customer-service\venv\lib\site-packages\flask\templating.py", line 149, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "C:\Users\zayaz\PycharmProjects\Customer-service\venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\zayaz\PycharmProjects\Customer-service\venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\Users\zayaz\PycharmProjects\Customer-service\venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\zayaz\PycharmProjects\Customer-service\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\Users\zayaz\PycharmProjects\Customer-service\venv\lib\site-packages\flask\templating.py", line 59, in get_source
    return self._get_source_fast(environment, template)
  File "C:\Users\zayaz\PycharmProjects\Customer-service\venv\lib\site-packages\flask\templating.py", line 95, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: user.html
2022-06-23 23:03:16,362 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 23:03:18,408 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 23:03:49,856 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 23:04:11,296 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:78]
2022-06-23 23:10:16,951 INFO: All good [in C:\Users\zayaz\PycharmProjects\Customer-service\app\__init__.py:79]
